document/FileFormat.mdk

```diff
@@ -1908,7 +1915,7 @@ namespace A {
 
 (These could not coexist in the same compilation, but there is no reason two such source files could not exist.)
 
-If the tool detected one result in class `B` in file1.cpp, and another result in namespace `B` in file2.cpp,
+If the tool detected one result in class `B` in _file1.cpp_, and another result in namespace `B` in _file2.cpp_,
 the `fullyQualifiedLogicalName` for both would be `A::B`.
 In that case, the tool might set the `logicalLocationKey` property in either one of the results to `A::B-0`,
```

Comment:
Make this A::B-1, as in your rule key example above. Presumably, A::B already exists, so we assume that's the 0th item. :)